project-base/tests/ShopBundle/Functional/Model/Cart/Watcher/CartWatcherTest.php

```diff
@@ -43,7 +43,7 @@ public function testGetModifiedPriceItemsAndUpdatePrices()
         $modifiedItems1 = $cartWatcher->getModifiedPriceItemsAndUpdatePrices($cart);
         $this->assertEmpty($modifiedItems1);
 
-        $pricingGroup = $this->getReference(PricingGroupDataFixture::PRICING_GROUP_ORDINARY_DOMAIN_1);
+        $pricingGroup = $this->getReferenceForDomain(PricingGroupDataFixture::PRICING_GROUP_ORDINARY, 1);
```

Comment:
Can we avoid using hardcoded `1`? Rather extract it into a variable or a constant please.

Reply:
I guess @DavidOstrozlik already mentioned it somewhere. I am just reminding it.

Reply:
We always have in our projects class `DomainHelper` that contains constants for domains. It would be nice to have this class in project-base by default.

Reply:
Do you really want me to change all the usages of `1` in tests with a constant? I am perfectly ok with hardcoding it...

Reply:
Putting it in constatn would be nice. In this tests is `1` user as `domainId`, `expoctedCount`, `quantity`. It is not a big pain here and now, but naming these magic numbers is a good practice in bigger scopes.

https://refactoring.guru/replace-magic-number-with-symbolic-constant

Reply:
bb7697697